geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/sortedset/AbstractZAddIntegrationTest.java

```diff
@@ -152,18 +152,72 @@ public void zaddCountsOnlyNewMembers_givenMultipleCopiesOfTheSameMember() {
 
   @Test
   public void zaddCountsOnlyNewMembers_givenMultipleCopiesOfTheSameMember_toAnExistingSet() {
-    Long addCount = (Long) jedis.zadd(SORTED_SET_KEY, 1.0, "otherMember");
+    Long addCount = jedis.zadd(SORTED_SET_KEY, 1.0, "otherMember");
     assertThat(addCount).isEqualTo(1);
     jedis.sendCommand(SORTED_SET_KEY, Protocol.Command.ZADD, SORTED_SET_KEY,
         "1", "member", "2", "member", "3", "member");
     assertThat(jedis.zcard(SORTED_SET_KEY)).isEqualTo(2);
     assertThat(jedis.zscore(SORTED_SET_KEY, "member")).isEqualTo(3.0);
   }
 
+  @Test
+  public void zaddDoesNotCountExistingMembersWithoutChanges_whenCHSpecified() {
+    Map<String, Double> initMap = makeMemberScoreMap(2 * INITIAL_MEMBER_COUNT, 0);
```

Comment:
Can the initial size be just `INITIAL_MEMBER_COUNT` for consistency? It seems arbitrary to double it in these tests.